experiment/builder_runner.py

```diff
@@ -227,9 +237,16 @@ def build_target_local(self,
         '-v',
         f'{outdir}:/out',
         '-v',
-        f'{get_workdir(generated_project)}:/work',
+        f'{workdir}:/work',
+        # Allows jcc to write err.log.
+        # From https://github.com/google/oss-fuzz/blob/090e0d6/infra/base-images/base-builder/jcc/jcc.go#L360
+        '-v',
+        f'{workspacedir}:/workspace',
```

Comment:
I recall you used to mount both `workspacedir` and `outdir` to `/out`, does it have any defects?

Reply:
Doing that makes /workspace mirrors /out in docker. There wasn't any defect observed since we never remove these dirs and no file has duplicated names, but I think it's better to just make another local dir